src/main/java/com/google/cloud/spanner/jdbc/JdbcResultSet.java

```diff
@@ -576,6 +588,7 @@ private Object getObject(Type type, int columnIndex) throws SQLException {
     if (type == Type.int64()) return getLong(columnIndex);
     if (type == Type.numeric()) return getBigDecimal(columnIndex);
     if (type == Type.string()) return getString(columnIndex);
+    if (type == Type.json()) return getString(columnIndex);
```

Comment:
I think this is the way to go, but more of a design question: do we ever want to externalise `Value.json` as a return type as well?

Reply:
Good point.

What we could do is modify the implementation of this method: https://github.com/googleapis/java-spanner-jdbc/blob/4ed17f0061077505d772c89589a0965f3bbc4a18/src/main/java/com/google/cloud/spanner/jdbc/JdbcResultSet.java#L896

It currently supports converting between common Java types, e.g. converting a `Timestamp` to a `String`. We could add support for specifying `com.google.cloud.spanner.Value.class` as the type, and then return the value in the result set as a `Value`.

Reply:
That makes sense to me. Does not necessary need to be a change added to this PR if you feel it should be out of scope here.

Reply:
I think that should be a separate change. I'll create a PR specifically for that, and once that is merged, add it for JSON here as well.